org.kde.ktechlab.json

```diff
@@ -0,0 +1,64 @@
+{
+   "id": "org.kde.ktechlab",
+   "runtime": "org.kde.Platform",
+   "runtime-version": "5.15",
+   "sdk": "org.kde.Sdk",
+   "command": "ktechlab",
+   "finish-args": [
+      "--share=ipc",
+      "--socket=fallback-x11",
+      "--socket=wayland",
```

Comment:
`Qt: Session management error: Could not open network socket`. 
Add `"--share=network",`

Reply:
I don't think that's necessary.

Reply:
It's not necessary, but it's nicer not to have spurious warnings

Reply:
Which is something the user never sees. Unless the application uses network,  it doesn't need access to it.

Reply:
The default instructions in flathub.org are to use the command line to install and run.